src/Apps/Debug/debugRoutes.tsx

```diff
@@ -26,13 +35,11 @@ export const debugRoutes: RouteProps[] = [
     children: [
       {
         path: "baseline",
-        // layout: "LogoOnly",
-        Component: DebugApp,
+        Component: DebugBaselineRoute,
```

Comment:
@dzucconi - part of the issue is that this isn't using `getComponent`, which invokes a promise and executes only when matching the route. Right now this loads the bundle statically, at boot.  Ditto elsewhere.

Reply:
https://github.com/artsy/force/blob/main/src/Apps/Artwork/artworkRoutes.tsx#L18